src/Popover/Popover.js

```diff
@@ -130,14 +130,16 @@ class Popover extends Component {
     this.handleResize = throttle(this.setPlacement, 100);
     this.handleScroll = throttle(this.setPlacement.bind(this, true), 50);
 
+    this.popoverRefs = {};
+
     this.state = {
       open: props.open,
       closing: false,
     };
   }
 
   componentDidMount() {
-    this.setPlacement();
+    setTimeout(this.setPlacement, 1);
```

Comment:
That's hidden something, I would normally ask to look into it, we shouldn't have such code. But anyway, the `master` is no longer a priority. Also, that might lead to other potential edge cases.

Can we just clear the timeout when unmounting? Also, no need to set `1`, `0` or no argument is enough.